chroma-manager/Makefile

```diff
@@ -149,7 +149,7 @@ agent_tests:
 			| tee ../chroma-agent/unit.log; \
 	exit $${PIPESTATUS[0]}
 
-unit_tests:
+unit_tests unit-tests:
```

Comment:
So that one can say "make unit-tests" if that is what one is used to.  It's causing no harm.